src/components/Page_404/Error.css

```diff
@@ -0,0 +1,90 @@
+*{
+margin:0;
+padding:0;
+background-color:white;
+width: 100% vw;
+height: 100% vh;
```

Comment:
Don't add background-color:white to the wildcard selector. It will affect other components .
The value 100% vw or 100% vh is not valid and it's better you don;t give them to wildcard selector as well.
Make the changes.